### PR TITLE
Parachain governance

### LINKED ISSUE #127: Allow Technical Committee To Execute Sudo Extrinsics
# Problem

Currently, within the Kusama parachain, there exists a solitary account with the authority to execute sudo transfers, including those linked to the establishment of HRMP channels.

# Solution

A reimplementation is essential, aligning with the mainnet structure, where extrinsics necessitating sudo privileges can be executed following the approval by the technical committee.

# DoD

- [ ] Technical committee implemented the same way as on mainnet
- [ ] SUDO access is revoked from all current standalone accounts.

### LINKED ISSUE #128: [Tech]: Implement Council (Governance)
### Problem statement

We can currently only conduct governance operation through the sudo account. We need to implement the Council on the Kusama parachain, the same way it is implemented on the mainnet.

### Description

_No response_

### Definition of Done

- [ ] Council implemented as on the mainnet

### Requirements

_No response_

### PR DESCRIPTION
**Pull request by N1ghtStorm**

### CODE REVIEW

File: parachain-common/src/weights.rs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+use sp_arithmetic::Perbill;
+
+use crate::primitives::Balance;
+use frame_support::dispatch::DispatchClass;
+
+/// We allow `Normal` extrinsics to fill up the block up to 75%, the rest can be used
+/// by  Operational  extrinsics.
+const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
+/// We allow for 2 seconds of compute with a 6 second average block time.
+const MAXIMUM_BLOCK_WEIGHT: Weight =
```
Comment (vovac12) [inline parachain-common/src/weights.rs]: We already have this types in runtime/src/lib.rs. And also 2 secs is too much for parachain, it's usually 0.5sec AFAIK

File: runtime/src/impls.rs [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+        // We want to keep it as high as possible, but can't risk having it reject,
+        // so we always the base block execution weight as a max
+        max_dispatch_weight.min(weight)
+    }
+    fn disapprove_proposal(p: u32) -> Weight {
+        <() as pallet_collective::WeightInfo>::disapprove_proposal(p)
+    }
+}
+
+impl pallet_democracy::WeightInfo for DemocracyWeightInfo {
```
Comment (vovac12) [inline runtime/src/impls.rs]: I think this impl can be removed here and in mainnet

**Review by vovac12 — CHANGES_REQUESTED** [review]

File: parachain-common/src/weights.rs (hunk 1 above)
Comment (N1ghtStorm) [reply parachain-common/src/weights.rs]: changed to default MAXIMUM_BLOCK_WEIGHT on the parachain

File: runtime/src/impls.rs (hunk 2 above)
Comment (N1ghtStorm) [reply runtime/src/impls.rs]: done

**Review by vovac12 — APPROVED** [review]

**Review by vovac12 — APPROVED** [review]

**Pull request merged**